### DISCUSSION TITLE
British National Grid Projection

### DISCUSSION THREAD

**Discussion by mfgt84** [opengisch/QField#3236] [Q&A]
We have an issue with the positional accuracy in QField compared to Zeno Mobile. On a test, using a known point, a point captured in Zeno Mobile landed exactly on top of the Ordnance Survey point as expected - reporting an accuracy (with RTK) of 2cm. Switching software to QField and taking the same point yielded a geometry around 1.5m away from this. Having tested it at various points, it seems to be a consistent offset issue. With QField also reporting good accuracy, I can only assume that it is a projection issue. 
I should say that the ESRI Mobile App produces a similar point to QField...
In the Zeno Mobile app, you have to pick a transformation method of 'OSGB Surveying' which I believe is based on the OSTN15 (https://www.ordnancesurvey.co.uk/business-government/tools-support/os-net/for-developers). I thought QField probably used this transformation by default anyway. But perhaps not.
Is this something anyone else has had an issue with and can shed any light on it for me?

**Comment by coastalrocket** [comment]
I'd imagine you'll need to set up the transformation you're after in your QGIS project. You'll need to have the OSTN15_NTv2_OSGBtoETRS.gsb file available to QGIS. Once you've got it available to QGIS you can go to Options > Transformations > and setup a new default datum transformation. Select EPSG:4326 as source and EPSG:27700 as destination. I'd expect the QField plugin would move the .gsb file to a proj folder within your project as part of the packaging. If not perhaps try doing it manually. I haven't tested this but that's what I would expect the project structure to look like.

**Reply by m-kuhn** [reply]
You will need to put the same file onto the QField device, the reference doc is a bit short on this topic but here's the pointer: https://docs.qfield.org/how-to/gnss/#altitude-correction-vertical-grid-shift
